### PR TITLE
Update travis to just check against node 6 & 8

### PR DESCRIPTION
**Pull request by junedkazi**
@bigcommerce/storefront-team

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
   "devDependencies": {
     "code": "^4.0.0",
     "eslint": "^3.15.0",
-    "lab": "^13.1.0",
+    "lab": "^14.3.1",
```
Comment (junedkazi) [inline package.json]: need to fix memory leaks

File: .travis.yml [hunk 2]
```diff
@@ -1,10 +1,6 @@
 language: node_js
 sudo: false
 node_js:
-  - 4
-  - 5
-  - 6
```
Comment (mattolson) [inline .travis.yml]: This repo is still being used by apps that use Node 6, so we shouldn't yet remove it here.

File: package-lock.json [hunk 3]
```diff
@@ -2,26 +2,46 @@
   "name": "@bigcommerce/stencil-styles",
   "version": "1.1.0",
   "lockfileVersion": 1,
+  "requires": true,
   "dependencies": {
     "@bigcommerce/node-sass": {
-      "version": "git://github.com/bigcommerce-labs/node-sass.git#4ea0b8d5f7c418d5bf20b07e395b2609e020bb8b"
+      "version": "git://github.com/bigcommerce-labs/node-sass.git#4ea0b8d5f7c418d5bf20b07e395b2609e020bb8b",
```
Comment (mattolson) [inline package-lock.json]: I don't think we should have a package-lock.json file for this repository. It is a library included by other applications, so the dependencies declared in package.json should be enough. Let's just remove the file entirely.

File: .travis.yml (hunk 2 above)
Comment (junedkazi) [reply .travis.yml]: Since this is going to be a major release. So I was thinking about dropping node 6 support.

File: package-lock.json (hunk 3 above)
Comment (junedkazi) [reply package-lock.json]: `package-lock.json` exists for a while now. Should I remove it ?

Comment (mattolson) [reply package-lock.json]: Yes

File: .travis.yml (hunk 2 above)
Comment (mattolson) [reply .travis.yml]: Ok, as long as this is a major release.

**Comment by mattolson** [comment]
@junedkazi You have failing tests

**Pull request closed without merging**